core/src/main/java/com/datastrato/gravitino/storage/relational/converters/MySQLExceptionConverter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.storage.relational.converters;
+
+import com.datastrato.gravitino.Entity;
+import com.datastrato.gravitino.exceptions.AlreadyExistsException;
+import com.datastrato.gravitino.exceptions.GravitinoRuntimeException;
+import java.sql.SQLException;
+
+/** Exception converter to Gravitino exception for MySQL. */
+public class MySQLExceptionConverter implements SQLExceptionConverter {
+
+  @SuppressWarnings("FormatStringAnnotation")
+  @Override
+  public GravitinoRuntimeException toGravitinoException(
+      SQLException se, Entity.EntityType type, String name) {
+    switch (se.getErrorCode()) {
+      case 1062:
```

Comment:
Is `1062`  the value defined by MySQL internally which means a table or a database already exists?

Reply:
It means found a duplicated entry for a primary key or a unique key, see: https://dev.mysql.com/doc/connector-j/en/connector-j-reference-error-sqlstates.html.
![image](https://github.com/datastrato/gravitino/assets/26177232/ce008cfe-bd1a-4882-8f93-c62252b85f3f)

Reply:
OK

Reply:
Can you please attach the document links to the code here? @xloya

Reply:
> Can you please attach the document links to the code here? @xloya

Have added the document link in annotations.